kubeflow/tf-training/prototypes/tf-job-operator.jsonnet

```diff
@@ -13,6 +13,7 @@
 // @optionalParam injectIstio string false Whether to inject istio sidecar; should be true or false.
 // @optionalParam clusterDomain string cluster.local DNS config to cluster domain.
 // @optionalParam monitoringPort string 8443 Port for monitoring agent to scrape metrics from.
+// @optionalParam enablePrometheus string false If set true, enable prometheus monitoring metrics reporting.
```

Comment:
Since monitoring is enabled by default, does this flag add any extra meaning?

Reply:
Agree - I think we should just have this enabled. Also kustomize doesn't have the notion of boolean flags.